### PR TITLE
DIOS-2863 Add codecs in use to getstatsdb

### PR DESCRIPTION
**Pull request by bcostdolby**

### CODE REVIEW

File: lib/IncomingStreamTrack.js [hunk 1]
```diff
@@ -92,6 +92,7 @@ const {
  * @property {number} [targetWidth] signaled target width on the VideoLayersAllocation header
  * @property {number} [targetHeight] signaled target height on the VideoLayersAllocation header
  * @property {number} [targetFps] signaled target fps on the VideoLayersAllocation header
+ * @property {number} [codec] Last codec in use
```
Comment (murillo128) [inline lib/IncomingStreamTrack.js:95]: return a string and not a number, you can use `const char* GetNameForCodec(MediaFrame::Type media,DWORD codec)`
https://github.com/medooze/media-server/blob/master/include/codecs.h#L160C1-L160C71

File: src/RTPIncomingSourceGroup.i [hunk 2]
```diff
@@ -19,6 +19,7 @@ struct RTPIncomingSourceGroup : public RTPIncomingMediaStream
 	DWORD maxWaitedTime;
 	double avgWaitedTime;
 	QWORD lastUpdated;
+	DWORD codec;
```
Comment (murillo128) [inline src/RTPIncomingSourceGroup.i:22]: you can extend the swig grapper to add a new atribute that does the number<->string conversion automatically

**Review by murillo128 — CHANGES_REQUESTED** [review]

File: lib/IncomingStreamTrack.js (hunk 1 above)
Comment (mildsunrise) [reply lib/IncomingStreamTrack.js:95]: agree, for consistency with the rest of the api we sould use strings

Comment (bcostdolby) [reply lib/IncomingStreamTrack.js]: done

File: src/RTPIncomingSourceGroup.i (hunk 2 above)
Comment (bcostdolby) [reply src/RTPIncomingSourceGroup.i]: done

File: lib/IncomingStreamTrack.js [hunk 3]
```diff
@@ -92,6 +92,7 @@ const {
  * @property {number} [targetWidth] signaled target width on the VideoLayersAllocation header
  * @property {number} [targetHeight] signaled target height on the VideoLayersAllocation header
  * @property {number} [targetFps] signaled target fps on the VideoLayersAllocation header
+ * @property {string} [codec] Name of the codec last in use, media-server codecs.h for details
```
Comment (murillo128) [inline lib/IncomingStreamTrack.js:95]:
```suggestion
 * @property {string} [codec] Name of the codec last in use
```

**Review by murillo128 — APPROVED** [review]
lgtm

**Review by murillo128 — APPROVED** [review]
lgtm

**Pull request merged**